### PR TITLE
ddl: improve log messages

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Comment by tangenta** [comment]
Please also update the usage in following logs:
```
[ddl] run reorg job done
[ddl] switch metadata lock feature
[ddl] SwitchConcurrentDDL
[autoid client] AllocAutoID grpc error, reconnect
```

**Comment by hawkingrei** [comment]
> Please also update the usages in following logs:
> 
> ```
> [ddl] run reorg job done
> [ddl] switch metadata lock feature
> [ddl] SwitchConcurrentDDL
> [autoid client] AllocAutoID grpc error, reconnect
> ```

All have been updated.

File: ddl/index.go [hunk 1]
```diff
@@ -755,7 +755,11 @@ func IngestJobsNotExisted(ctx sessionctx.Context) bool {
 // tryFallbackToTxnMerge changes the reorg type to txn-merge if the lightning backfill meets something wrong.
 func tryFallbackToTxnMerge(job *model.Job, err error) {
 	if job.State != model.JobStateRollingback {
-		logutil.BgLogger().Info("[ddl] fallback to txn-merge backfill process", zap.Error(err))
+		if err != nil {
+			logutil.BgLogger().Warn("[ddl] fallback to txn-merge backfill process", zap.Error(err))
+		} else {
+			logutil.BgLogger().Info("[ddl] fallback to txn-merge backfill process")
+		}
```
Comment (tangenta) [inline ddl/index.go:762]: Sorry, I just found that this is the dead code.. Please remove this : )

Comment (hawkingrei) [reply ddl/index.go]: Done

**Review by tangenta — APPROVED** [review]

**Review by Defined2014 — APPROVED** [review]

**Pull request merged**